plugins/modules/aci_interface_config.py

```diff
@@ -0,0 +1,440 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Sabari Jaganathan <sajagana@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+
+DOCUMENTATION = r"""
+---
+module: aci_interface_config
+short_description: Manage the Port Configuration of the Fabric Access Policies - Interface Configuration (infra:PortConfig)
+description:
+- Manage the Port Configuration of the Fabric Access Policies - Interface Configuration on Cisco ACI fabrics.
+options:
+  policy_group:
+    description:
+    - The Name of the Policy Group.
```

Comment:
For consistency would think say this needs to change:

```suggestion
    - The name of the Policy Group.
```